docs/network-flow-visibility.md

```diff
@@ -170,6 +171,15 @@ kubectl delete -f flow-visibility.yml
 kubectl delete -f https://raw.githubusercontent.com/antrea-io/theia/main/build/charts/theia/crds/clickhouse-operator-install-bundle.yaml -n kube-system
 ```
 
+From Theia 0.2, Theia supports ClickHouse clustering in lieu of the non-clustered
```

Comment:
Would it better if we put this paragraph before the instructions of stopping Grafana Flow Collector? As it is required to be done before those. No strong opinion here, would like to hear your thoughts.

Reply:
I agree. Slightly rephrase the paragraph. Would you like to take another look if it is better now?

Reply:
LGTM